assets/company/store-appstore.svg

```diff
@@ -1,13 +1,37 @@
-  </g>
+<svg width="119px" height="35px" viewBox="0 0 119 35" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
That's correct. The problem with the Google Play image was the gradient was not displaying, due to id conflicts in the inlined SVG. https://github.com/Wikia/upstream/pull/733 resolves that. The Apple store image then had the problems described in this PR. So they are two separate issues.